__tests__/app.test.js

```diff
@@ -52,4 +52,43 @@ describe("GET /api/reviews", () => {
             })
         })
     })
+})
+
+describe("GET /api/reviews/:review_id", () => {
+    test("200: returns a review object with specified review_id", () => {
+        return request(app)
+        .get("/api/reviews/2")
+        .expect(200)
+        .then(( { body } ) => {
+            const review = body.review
+            expect(review).toBeInstanceOf(Object)
+            expect(review).toEqual(expect.objectContaining({
+                owner: expect.any(String),
+                title: expect.any(String),
+                review_id: expect.any(Number),
+                category: expect.any(String),
+                review_img_url: expect.any(String),
+                review_body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                designer: expect.any(String)
+            }))
```

Comment:
You can be more specific here, if you are going to reviews/2 then you should know what the the object coming back looks like. Instead of using expect.any(), use the actual values.